src/cart/src/cart/crt_hlc.c

```diff
@@ -47,6 +47,9 @@
 
 static ATOMIC uint64_t crt_hlc;
 
+/** See crt_hlc_epsilon_set's API doc */
+static uint64_t crt_hlc_epsilon = 100 * 1000 * 1000;
```

Comment:
Do you think 100ms is enough? I think this is too small for big cluster or slow network.

Reply:
So far 100 ms feels like a relatively conservative default to me. That is for configurations where all HLCs' physical clocks are synchronized using a LAN to a local head node, which in turn synchronizes to a remote NTP server. On Wolf, with chronyd, it's easy to make the epsilon among 5 nodes to be < 1 ms (actually < 200 us). You're right, however, I don't have any numbers from larger clusters yet.

I'm open to change the value. Do you have some number in mind?

Reply:
From my point of view it should be at least 1sec. If you just ping any node in our cluster through Ethernet you will get 300ms average time of ping. If network is overloaded this time will be more.

Reply:
The value does not have to larger than the network latency. That said, I'm fine with a conservative 1 second to begin with. We can refine it next.